src/qml/AddContactPage.qml

```diff
@@ -0,0 +1,71 @@
+/*
+ *  Kaidan - Cross platform XMPP client
+ *
+ *  Copyright (C) 2017 JBB <jbb.mail@gmx.de>
+ *
+ *  Kaidan is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Kaidan is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Kaidan. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import QtQuick 2.0
+import QtQuick.Controls 2.0 as Controls
+import QtQuick.Layouts 1.2
+import org.kde.kirigami 1.0 as Kirigami
+
+Kirigami.Page {
```

Comment:
I don't want to use the popup anymore, it produces problems with the drawer, is not the best on mobile and has to be centered in the middle of the window. The page looks not so great on desktop yet...